Products/CMFPlone/browser/icon.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+from OFS.Image import File
+from plone.registry.interfaces import IRegistry
+from Products.CMFCore.interfaces import ISiteRoot
+from zope.component import adapter
+from zope.component import getUtility
+from zope.component.hooks import getSite
+from zope.interface import implementer
+from zope.interface import Interface
+from zope.location.interfaces import LocationError
+from zope.traversing.interfaces import ITraversable
+from zope.publisher.interfaces import IPublishTraverse
+
+
+@implementer(IPublishTraverse)
+class IconView:
+
+    prefix = 'plone.staticresources.icon.'
```

Comment:
we should name it just `plone.icon.`

Reply:
This needs to fit to what we register in plone.staticresources. I agree to keep the name as short as possible.

Reply:
Then we should change it over there.